helm/macropower-analytics-panel-server/templates/ingress.yaml

```
@@ -0,0 +1,22 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ include "resource.default.name" . }}
+  namespace: {{ .Values.namespace }}
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/rewrite-target: /$2
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+spec:
+  rules:
+  - host: {{ .Values.server.ingress.host }}
+    http:
+      paths:
+      - backend:
+          service:
+            name: {{ include "resource.default.name" . }}
+            port:
+              number: {{ .Values.server.service.port }}
+          path: /{{ .Values.server.ingress.analytics }}(/|$)(.*)
```

This seens to have reverted:

```suggestion
          path: /{{ .Values.server.ingress.path }}(/|$)(.*)
```